src/WebJobs.Script/Binding/GeneralScriptBindingProvider.cs

```diff
@@ -0,0 +1,136 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Reflection;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Script.Description;
+using Microsoft.Azure.WebJobs.Script.Extensibility;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.Binding
+{
+    // General binder that works directly off SDK interfaces. 
+    class GeneralScriptBindingProvider : ScriptBindingProvider
+    {
+        public ITooling Tooling { get; set; }
```

Comment:
Since we're the only one that implement it; we can add new methods to the interface without it being a breaking change. 

---
In reply to: [108999816](https://github.com/Azure/azure-webjobs-sdk-script/pull/1349#discussion_r108999816) [](ancestors = 108999816)